src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ReplacementPlanRequest.cs

```diff
@@ -6,9 +6,9 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
     public class ReplacementPlanRequest
     {
         [Required]
-        public Guid? OriginalSubjectId { get; set; }
+        public Guid? OriginalReleaseFileReferenceId { get; set; }
```

Comment:
why are these optional?

Reply:
Because otherwise missing params in the Http request get coerced into Guid which has a default of `00000000-0000-0000-0000-000000000000`. The `Required` check for each field passes, and the model check is valid.

We then get replacement parameters of  `00000000-0000-0000-0000-000000000000` going into the service which we don't want.

So effectively without using Optional Guid's, the `Required` check is ignored because the Guids still get instantiated 

Optional Guids seems to be the right way to do it, where the value is either present with a valid Guid, or not set at all.